src/scripts/main.js

```diff
@@ -2,7 +2,49 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboardTable = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(person => {
+  const row = document.createElement('tr');
+
+  const nameCell = document.createElement('td');
+
+  nameCell.textContent = person.name;
+
+  const genderCell = document.createElement('td');
+
+  genderCell.textContent = (() => {
+    if (person.sex === 'm') {
+      return 'Male';
+    }
+
+    return 'Female';
+  })();
```

Comment:
let's just use ternary operator here instead of the IIFE

Reply:
You right, its looks better!